pkg/datastore/filedb/filedb.go

```diff
@@ -96,13 +95,7 @@ func (f *FileDB) Get(ctx context.Context, col datastore.Collection, id string, v
 		parts = append(parts, part)
 	}
 
-	if len(parts) == 1 {
-		return json.Unmarshal(parts[0], v)
-	}
-
-	// TODO: Add merge based on UpdatedAt field in case there are multiple parts of object are fetched.
```

Comment:
Please close this issue once this PR got merged.

Reply:
there is no issue like that, since I used the `/todo skip` command last time 🙏

Reply:
Thanks for your confirmation. 👍